gourmet/importers/plaintext_importer.py

```diff
@@ -45,32 +45,32 @@ def handle_line (self):
         raise NotImplementedError
 
     def compile_regexps (self):
-        self.blank_matcher = re.compile("^\s*$")
+        self.blank_matcher = re.compile(r"^\s*$")
         # out unwrap regexp looks for a line with no meaningful characters, or a line that starts in
         # ALLCAPS or a line that is only space. (we use this with .split() to break text up into
         # paragraph breaks.
-        self.unwrap_matcher = re.compile('\n\W*\n')
-        self.find_header_breaks_matcher = re.compile('\s+(?=[A-Z][A-Z][A-Z]+:.*)')
+        self.unwrap_matcher = re.compile(r'\n\W*\n')
+        self.find_header_breaks_matcher = re.compile(r'\s+(?=[A-Z][A-Z][A-Z]+:.*)')
 
     def unwrap_lines (self, blob):
         if blob.find("") >= 0:
```

Comment:
Can we please not use the non-printable here. I guess it's `\u2029` but it's anybody's guess.

Reply:
I think it's tab, disguised as `^T`, also on line 60. I checked it by opening the file with `vim -b plaintext_importer.py`

Reply:
Okay. This certainly doesn't match the comment. https://docs.python.org/3/library/stdtypes.html#str.splitlines talks about `\u2029` as "Paragraph Separator"

Reply:
Seems you're right

Reply:
Ok, I think it's this guy: https://www.fileformat.info/info/unicode/char/0014/index.htm

`print(repr("")) -> '\x14'`

Reply:
I can't find any reference anywhere that DeviceControlFour is used as paragraph seperator. I think we either use the unicode one (https://www.fileformat.info/info/unicode/char/2029/index.htm `\u2029`) or remove the code alltogether. It also does not appear anywhere else in the `gourmet` source.

Reply:
I suggest not to remove it just yet, and use `\u2029`. I don't have a sound argument for that, though.

Reply:
We have decided that the following would be good: replace the strange character with the proper Unicode one and then use splitlines, which recognizes the paragraph symbol. This should boil down more or less to a single line of  `paragraphs =oldstring.replace(u14, u1234).splitlines()`

That's good